### ISSUE TITLE
Comments are out of order

### ISSUE THREAD

**Issue by polsvoice** [diaspora/diaspora#4269]
I notice that, many times, comments on posts from JoinDiaspora show up out of order on Diasp.org (or are missing!) This makes it hard to follow conversations. Here is an example from [a recent post by Sean Tilley.](https://diasp.org/posts/2065030#)
Here's how the comments look on JoinDiaspora:
![diaspora_screenshot3](https://f.cloud.github.com/assets/941129/726565/68cced52-e10b-11e2-9e0a-ece723c71d21.jpg)
On diasp.org:
![diaspora_screenshot4](https://f.cloud.github.com/assets/941129/726566/68dfc2a6-e10b-11e2-9871-c9a3ec94497a.jpg)
May be related to [this issue?](https://github.com/diaspora/diaspora/issues/3350)
##

**Comment by Flaburgan** [comment]
@JannikStreek any news?

**Comment by cmrd-senya** [comment]
Won't #6406 resolve it? It sorts comments by the time stamps that presented in the DOM generated for every comment model.

**Comment by cmrd-senya** [comment]
Oh, I see, the time stamps are probably different across pods, that's the reason.

**Comment by SuperTux88** [comment]
Fixed with #7436
